cli/example-ifc.config.js

```diff
@@ -12,8 +16,12 @@ module.exports = function(frameRouter) {
       custom: getCustomClientData()
     }
   );
+
+  frameRouter.messaging.addListener('publish.topic', publication => {
+    hostActions.notifyPubSub(publication);
+  });
```

Comment:
I set up this `notifyPubSub` hook because the host app can't know in advance what topics the client will send, but after thinking about it this morning, I think rather than providing a hook to show pubSub events, it would be better for the setup function to just return some config like: 
```
{
   clientTopics: [ 'publish.topic' ]
}
```